lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -87,6 +92,35 @@ class SessionStateEnforcer {
    */
   bool set_upf_node(const std::string& node_id, const std::string& n3_addr);
 
+  /* Send session requst to upf */
+  void m5g_send_session_request_to_upf(
+      const std::string& imsi, const std::unique_ptr<SessionState>& session);
+
+  /*
+   * To send session or UE notifcaiton
+   * to AMF, this cud be session state or
+   * paging notification
+   */
+  void handle_state_update_to_amf(
+      SessionState& session_state, const magma::lte::M5GSMCause m5gsmcause,
+      NotifyUeEvents event);
+
+  /* Pdr State change routine */
+  void m5g_pdr_rules_change_and_update_upf(
+      const std::string& imsi, const std::unique_ptr<SessionState>& session,
+      enum PdrState pdrstate);
+
+  /*Start processing to terminate respective session requested from AMF*/
+  void m5g_start_session_termination(
+      SessionMap& session_map, const std::string& imsi,

```

Comment:
let's omit `imsi` from the parameters since we can easily get is like `session->get_imsi()`

Reply:
Removed the parameter imsi and used session->get_imsi() as suggested.